internal/tiltfile/docker.go

```diff
@@ -15,10 +15,7 @@ import (
 	"github.com/windmilleng/tilt/pkg/model"
 )
 
-const fastBuildDeprecationWarning = "FastBuild (`fast_build`; `add_fast_build`; " +
-	"`docker_build(...).add(...)`, etc.) will be deprecated **ON THURS. 9/12/19**; " +
-	"you can use Live Update instead! See https://docs.tilt.dev/live_update_tutorial.html " +
-	"for more information. If Live Update doesn't fit your use case, let us know."
+var fastBuildDeletedErr = fmt.Errorf("fast_build is deleted; check out live_update (e.g. https://docs.tilt.dev/live_update_tutorial.html")
```

Comment:
at the risk of getting soapboxy:

This message feels very "we did a thing, deal with it."-y, which, as a user, gives me low confidence that tool maintainers are invested in / aware of the pain their users experience.

how about: "Sorry, fast_build is no longer supported. This functionality is now provided via live_update: https://docs.tilt.dev/live_update_tutorial.html. If you run into problems, let us know: https://tilt.dev/contact"?